divintseg/_core.py

```diff
@@ -589,6 +589,57 @@ def isolation(
     return final_df
 
 
+def bells(
+    df_communities: pd.DataFrame,
+    group_name: str,
+    by: str,
+    over: str,
+) -> pd.DataFrame:
+    """
+    Computes the isolation of a group using the isolation index by Wendell Bell.
+
+    Parameters
+    ----------
+    df_communities
+        A :py:class:`pd.DataFrame` of communities.
+    group_name
+        The name of the group (name of a column in `df_communities`)
+        whose isolation we wish to compute.
+    by
+        The column or index to group by in order to
+        partition the rows into communities.
+    over
+        The column to group by in order to partition
+        the rows of each community into smaller
+        aggregation units where the base diversity will
+        be computed.
+
+    Returns
+    -------
+        A dataframe with one row for each unique value of the `by`
+        column indicating the Bell's Index of the `group_name` column
+        with respect to all of the other columns in the data frame.
```

Comment:
I would just add a comment here explaining that 1 is returned in the homogeneous case.